yammbs/_minimize.py

```diff
@@ -182,7 +182,11 @@ def _run_openmm(
     context.setPositions(
         (positions * openmm.unit.angstrom).in_units_of(openmm.unit.nanometer),
     )
-    openmm.LocalEnergyMinimizer.minimize(context, 5.0e-9, 1500)
+    openmm.LocalEnergyMinimizer.minimize(
+        context=context,
+        tolerance=10,
+        maxIterations=0,
+    )
```

Comment:
These are the current defaults (8.x line): http://docs.openmm.org/latest/api-python/generated/openmm.openmm.LocalEnergyMinimizer.html?highlight=localenergyminimizer#openmm.openmm.LocalEnergyMinimizer.minimize